press/agent.py

```diff
@@ -1110,6 +1110,15 @@ def get_cached_apps(self):
 			data={},
 		)
 
+	def get_site_apps(self, site):
+		raw_apps_list = self.get(
+			f"benches/{site.bench}/sites/{site.name}/apps",
+		)
+		apps: list[str] = [
```

Comment:
if previous variable is typed, this is not required, the type checker can impute it